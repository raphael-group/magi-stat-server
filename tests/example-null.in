{
    "X": [
        "Not mutated", 
		"null",
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
		"Mutated",
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
        "Mutated", 
        "Not mutated", 
        "Not mutated", 
        "Mutated", 
		"alien",
		"Mutated",
		"alien",
        "Not mutated"
    ], 
    "Y": [
        "male", 
        "alien", 
        "male", 
        "male", 
        "female", 
		"null",
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "female", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "female", 
        "male", 
        "female", 
        "female", 
        "male", 
        "female", 
        "female", 
        "female", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "female", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "female", 
        "female", 
        "female", 
        "female", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "female", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "female", 
        "female", 
        "female", 
        "male", 
        "female", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "female", 
        "female", 
        "female", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "male", 
        "male", 
        "male", 
        "male", 
        "male", 
        "female", 
        "female", 
        "male", 
        "male", 
 		"null",
		"null",
		"null",
		"female"
    ]
}