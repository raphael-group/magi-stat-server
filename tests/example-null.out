{
    "request": "contingency", 
    "samplesRemoved": 5, 
    "stats": {
        "Fisher's exact test, Mutated vs Not mutated on female vs male membership": {
            "X": "Mutated, Not mutated", 
            "Y": "female, male", 
            "p": 0.5133, 
            "report": {
                "html": "<b>P-value</b> = 0.513287", 
                "tex": "$P = 0.513287$"
            }
        }
    }, 
    "table": [
        [
            "", 
            "female", 
            "male"
        ], 
        [
            "Mutated", 
            7.0, 
            17.0
        ], 
        [
            "Not mutated", 
            96.0, 
            165.0
        ]
    ]
}
